deploy/handler/operator.yaml

```diff
@@ -56,38 +56,85 @@ spec:
               value: "False"
             - name: PROFILER_PORT
               value: "6060"
-            - name: CA_ROTATE_INTERVAL
-              value: {{ .CARotateInterval | default "8760h0m0s" }}
-            - name: CA_OVERLAP_INTERVAL
-              value: {{ .CAOverlapInterval | default "24h0m0s" }}
-            - name: CERT_ROTATE_INTERVAL
-              value: {{ .CertRotateInterval | default "4380h0m0s" }}
-            - name: CERT_OVERLAP_INTERVAL
-              value: {{ .CertOverlapInterval | default "24h0m0s" }}
           ports:
-          - containerPort: 8443
+          - containerPort: 9443
             name: webhook-server
             protocol: TCP
-          readinessProbe:
-            httpGet:
-              path: /readyz
-              port: webhook-server
-              scheme: HTTPS
-              httpHeaders:
-              - name: Content-Type
-                value: application/json
-            initialDelaySeconds: 10
-            periodSeconds: 10
           volumeMounts:
           - name: tls-key-pair
             readOnly: true
-            mountPath: /etc/webhook/certs
+            mountPath: /tmp/k8s-webhook-server/serving-certs/
       volumes:
         - name: tls-key-pair
           secret:
             secretName: {{template "handlerPrefix" .}}nmstate-webhook
 ---
 apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{template "handlerPrefix" .}}nmstate-cert-manager
+  namespace: {{ .HandlerNamespace }}
+  labels:
+    app: kubernetes-nmstate
+    component: kubernetes-nmstate-cert-manager
+spec:
+  replicas: 2
```

Comment:
nmstate already has quite a lot of moving parts. We don't mind downtime of couple of minutes in case one of the nodes running this goes down, do we? If not, could we keep it on a single replica? To save memory and CPU.